### ISSUE TITLE
allow multiple public-repos to be configured

### ISSUE THREAD

**Issue by duncdrum** [eXist-db/existdb-packageservice#16]
I have tried to point to multiple public repos but don't seem to get it to work as i expect. the following `configuration.xml`
```xml
<repositories>    
    <repository active="true" default="true">https://exist-db.org/exist/apps/public-repo</repository>
    <repository active="true" default="false">http://heml.ddns.net:8080/exist/apps/public-repo/</repository>
</repositories>
```
## expected behavior
should make all the apps from exist's public-repo (58) available plus the ones from the private repo. 

## actual behavior
no matter what combination of flags i set, only one public repo is available at a given time.

**Comment by adamretter** [comment]
<img width="645" alt="Screenshot 2020-10-03 at 17 03 38" src="https://user-images.githubusercontent.com/1264057/94994810-7458a100-059a-11eb-98b9-8e8be525e62e.png">

**Comment by duncdrum** [comment]
@adamretter that the repo in the example was down, is a separate issue. But as described in the op, I couldn’t get two repos to function in parallel irrespective of their html status. The service only pulls packages from one repo at a time

**Comment by duncdrum** [comment]
ok here is a reproducer `configuration.xml`:

```xml
<settings>
    <repositories>
        <repository active="true" default="true">https://www.duncdrum.eu/exist/apps/public-repo/</repository>
        <repository active="true" default="false">https://exist-db.org/exist/apps/public-repo/</repository>
    </repositories>
    <readonly>
        <package>http://exist-db.org/apps/existdb-packageservice</package>
        <package>http://exist-db.org/apps/packagemanager</package>
    </readonly>
</settings>
```

should show 59 available packages (my own repo just has a single test app)
instead 
![Screenshot 2020-10-05 at 16 43 05](https://user-images.githubusercontent.com/6205362/95094166-e4496180-0729-11eb-9c0e-7b8fbcdb28a5.png)
